src/analyses/mutexEventsAnalysis.ml

```diff
@@ -0,0 +1,153 @@
+(** Mutex analysis. *)
+
+module M = Messages
+module Addr = ValueDomain.Addr
+module Lockset = LockDomain.Lockset
+module Mutexes = LockDomain.Mutexes
+module LF = LibraryFunctions
+open Prelude.Ana
+open Analyses
+open GobConfig
+
+let big_kernel_lock = LockDomain.Addr.from_var (Goblintutil.create_var (makeGlobalVar "[big kernel lock]" intType))
+let console_sem = LockDomain.Addr.from_var (Goblintutil.create_var (makeGlobalVar "[console semaphore]" intType))
+let verifier_atomic = LockDomain.Addr.from_var (Goblintutil.create_var (makeGlobalVar "[__VERIFIER_atomic]" intType))
+
+module Spec: MCPSpec =
+struct
+  include UnitAnalysis.Spec
+  let name () = "mutexEvents"
+
+
+  (* NB! Currently we care only about concrete indexes. Base (seeing only a int domain
+     element) answers with the string "unknown" on all non-concrete cases. *)
```

Comment:
Is this comment still up-to-date? Where does Base answer with the string(?!) `unknown`?

Reply:
I don't think we do indeed, unless there's somehow actually `unknown_exp` in an index. I just removed that sentence.